src/utilities/api/morg_http_client.py

```diff
@@ -148,7 +149,223 @@ def get_is_player_idle(self) -> Union[bool, None]:
                 print(e)
                 return None
 
-            if data["animation"] != -1 or data["animation pose"] not in [808, 813]:
+            if data["animation"] != animation_ids.IDLE or data["animation pose"] not in [808, 813]:
+                return False
+        return True
+    
+    def get_is_player_woodcutting(self) -> Union[bool, None]:
```

Comment:
Each of those methods contains more or less the same code. I think it could be replaced by something like `get_is_player_doing(x)' where x is any list in animation_ids.

Reply:
I agree, it can be combined into a single function.